integration-tests/test_extensions.py

```diff
@@ -9,17 +9,20 @@ def test_serverextensions():
     # jupyter-serverextension writes to stdout and stderr weirdly
     proc = subprocess.run(
         ["/opt/tljh/user/bin/jupyter-server", "extension", "list", "--sys-prefix"],
-        stderr=subprocess.PIPE,
+        capture_output=True,
+        text=True,
     )
 
+    output = proc.stdout + proc.stderr
```

Comment:
Thanks for this change! It would be great to fix the integration tests.

One suggestion -- instead of using `capture_output` and concatenating the string objects, you could also set `stderr=subprocess.stdout` and check `assert 2 in proc.stdout.decode()`. 

In any case, this is mergeable, and I like the new assert string.